CriticalMassTests/CriticalMaps.xctestplan

```diff
@@ -0,0 +1,44 @@
+{
+  "configurations" : [
+    {
+      "id" : "3B90BF13-D62B-459B-B9C6-5162A3DEF527",
+      "name" : "Default",
+      "options" : {
+
+      }
+    },
+    {
+      "id" : "BF44ED31-49FD-48E0-AD35-4B279312F880",
+      "name" : "Configuration 2",
+      "options" : {
+
+      }
+    },
+    {
+      "id" : "FF294556-426B-40CD-A500-D4998EA2E3B9",
+      "name" : "Configuration 3",
+      "options" : {
+
+      }
```

Comment:
`Configuration 2` and `Configuration 3` have no different setting than `Default` (as I can see). The unit tests are currently run three times with these settings. I think these two additional configurations can be deleted. 

```suggestion
```